README.md

```diff
@@ -221,6 +221,8 @@ react_component('HelloMessage', {name: 'John'}, {prerender: true})
 ```
 This will return the fully rendered component markup, and as long as you have included the `react_ujs` script in your page, then the component will also be instantiated and mounted on the client.
 
+For additional debugging, you can optionally include `replay_console: true` in the options hash to replay any captured console messages from server-rendering back on the client.
+
```

Comment:
You can expand on saying, we are just adding as comments instead of actually running the console calls.